content/src/types.ts

```diff
@@ -64,6 +65,7 @@ export type AppComponents = {
   catalystFetcher: Fetcher
   daoClient: DAOClient
   server: Server
+  contentFolderMigrationManager: ContentFolderMigrationManager
```

Comment:
Please convert the MigrationManager into a function. It is not a port nor component nor manages anything